### PR TITLE
Disk configurable, enter path with a little helper in config

### PR DESCRIPTION
**Pull request by lockenkop**
#64 

I made the path that this Entity reads configurable.

image
image

I don't know how psutil.disk_partitions() and psutil.disk_usage() behaves in macos.
Might get a macos VM going.

### CODE REVIEW

**Comment by richibrics** [comment]
Great, I wanted to add this for a long time !

**Comment by richibrics** [comment]
Example from BigSur:
```
Python 3.11.3 (v3.11.3:f3909b8bc8, Apr  4 2023, 20:12:10) [Clang 13.0.0 (clang-1300.0.29.30)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import psutil
>>> psutil.disk_partitions() 
[sdiskpart(device='/dev/disk1s5s1', mountpoint='/', fstype='apfs', opts='ro,local,rootfs,dovolfs,journaled,multilabel', maxfile=255, maxpath=1024), sdiskpart(device='/dev/disk1s4', mountpoint='/System/Volumes/VM', fstype='apfs', opts='rw,noexec,local,dovolfs,dontbrowse,journaled,multilabel,noatime', maxfile=255, maxpath=1024), sdiskpart(device='/dev/disk1s2', mountpoint='/System/Volumes/Preboot', fstype='apfs', opts='rw,local,dovolfs,dontbrowse,journaled,multilabel', maxfile=255, maxpath=1024), sdiskpart(device='/dev/disk1s6', mountpoint='/System/Volumes/Update', fstype='apfs', opts='rw,local,dovolfs,dontbrowse,journaled,multilabel', maxfile=255, maxpath=1024), sdiskpart(device='/dev/disk1s1', mountpoint='/System/Volumes/Data', fstype='apfs', opts='rw,local,dovolfs,dontbrowse,journaled,multilabel', maxfile=255, maxpath=1024), sdiskpart(device='/dev/disk0s4', mountpoint='/Volumes/Untitled', fstype='ntfs', opts='ro,local,ignore-ownership,multilabel', maxfile=255, maxpath=1024)]
>>> psutil.disk_usage("/")
sdiskusage(total=274999996416, used=15322058752, free=27211255808, percent=36.0)
```

**Comment by lockenkop** [comment]
Looks good to me.

**Comment by richibrics** [comment]
Good. Do you plan adding also Windows support ?

**Comment by lockenkop** [comment]
I just got my Windows dev VM up and running. windows support for psutil.disk_* looks good.
I'll cook something up.

**Comment by lockenkop** [comment]
Got Windows working.
image
image

Had to do some reorganisation to make the config os agnostic.

**Review by infeeeee — APPROVED** [review]
Very nice! Change the default win path, and I think it can be pulled.

File: IoTuring/Entity/Deployments/Disk/Disk.py [hunk 1]
```diff
@@ -1,23 +1,99 @@
...
+CONFIG_KEY_DU_PATH = "path"
 
 
 class Disk(Entity):
+
     NAME = "Disk"
+    ALLOW_MULTI_INSTANCE = True
+    CONFIG_QUESTION = "which Drive shall be checked"
+    DEFAULT_PATH_UNIX = "/"
+    DEFAULT_PATH_WINDOWS = "C://"
```
Comment (infeeeee) [inline IoTuring/Entity/Deployments/Disk/Disk.py:18]:
```suggestion
    DEFAULT_PATH_WINDOWS = "C:\\"
```

Win uses backslash

**Comment by richibrics** [comment]
Thank you for your work @lockenkop !
Contributions are always appreciated 🤝

**Pull request merged**